partner_company_type/migrations/10.0.1.0.1/post-migration.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2017 ACSONE SA/NV
```

Comment:
If we still reed the rename of the field, we do not need it here. You can use the old_name= keyword on the field definition.